projects/core/src/product/occ/product-reviews.service.ts

```diff
@@ -0,0 +1,61 @@
+import { Injectable } from '@angular/core';
+import { HttpClient, HttpHeaders } from '@angular/common/http';
+
+import { Observable, throwError } from 'rxjs';
+import { catchError } from 'rxjs/operators';
+
+import { ReviewList, Review } from '../../occ/occ-models/occ.models';
+import { DynamicTemplate } from '../../config/utils/dynamic-template';
+
+import { OccProductConfig } from './product-config';
+import { ProductOccService } from './product-occ.service';
+
+@Injectable()
+export class OccProductReviewsService extends ProductOccService {
+  constructor(
+    private http: HttpClient,
+    private config: OccProductConfig,
+    private dynamicTemplate: DynamicTemplate
+  ) {
+    super(config);
+  }
+
+  protected getEndpoint(code: string, maxCount?: number): string {
```

Comment:
small detail, but what about moving the public classes to the top of the class? Same with other services